upup/pkg/fi/cloudup/template_functions.go

```diff
@@ -217,6 +217,14 @@ func (tf *TemplateFunctions) AddTo(dest template.FuncMap, secretStore fi.SecretS
 
 	dest["UseServiceAccountIAM"] = tf.UseServiceAccountIAM
 
+	if cluster.Spec.NodeTerminationHandler != nil {
+		dest["DefaultQueueName"] = func() string {
+			s := strings.Replace(tf.ClusterName(), ".", "-", -1)
+			url := "https://sqs." + tf.Region + ".amazonaws.com/" + tf.AWSAccountID + "/" + s + "-nth"
```

Comment:
This will work for commercial and govcloud regions but not for china. see https://docs.amazonaws.cn/en_us/general/latest/gr/sqs-service.html

I'm not sure of a better way to get this URL unfortunately.

Reply:
good call out. how about a regex on the region? 
something like:
```
domain := ".amazonaws.com/"
matched, err := regexp.MatchString(`^cn-`, tf.Region)
if err != nil {
  return ...
}
if matched {
  domain = ".amazonaws.com.cn/
}
...
url := "https://sqs." + tf.Region + domain + tf.AWSAccountID + "/" + s + "-nth"
```

Reply:
or strings.contains could possibly work as well
```
domain := ".amazonaws.com/"
if (strings.Contains(tf.Region, "cn-")) {
  domain = ".amazonaws.com.cn/
}
```